updating the MuscleTorquesWithBspline BC to be compatible with pyelas…

nmnaughton opened the pull request:
The numba/numpy import check was removed in the update so I removed the numpy version of the BC on the main branch. The repo should now work with the current version of pyelastica.

Commit:
updating the MuscleTorquesWithBspline BC to be compatible with pyelastica>0.2.0

armantekinalp reviewed (commented):
Looks good to me. 
But I have three suggestions
1.  Changing file names of `muscle_torques_with_bspline_numba.py` to `muscle_torques_with_bspline.py`
2. Moving files inside the `BsplineMusclesTorques` folder to one directory up (into `MuscleTorquesWithBspline` ) and removing `BsplineMusclesTorques`
3. Also what is this `Icon` file located inside the `MuscleTorquesWithBspline`. Do we need that? Can we remove?

Commit:
cleaning up the muscle import filenames and deleting icon files

nmnaughton commented:
I changed the names to `muscle_torques_with_bspline.py` as you suggested and deleted the Icon files. Not sure what those are and they aren't necessary. 

I did not implement suggestion 2. You are right that it would be cleaner, but it would break the import of any script written using the old version and removing one subfolder doesn't seem worth the hassle it if messes up anyone's script.

armantekinalp commented:
Okay sounds good to me. We can merge
